tests/20-SideMenu/BoardsMenuSection.test.tsx

```diff
@@ -26,62 +27,77 @@ describe("Regular", () => {
   test("Renders section with board menu items", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByText(Regular!.args!.title!)).toBeVisible;
+
+    const boards = screen.getAllByRole("link");
+    expect(boards).toHaveLength((Regular.args as BoardsSectionProps).boards!.length);
+    boards.forEach((board, i) => {
+      expect(board).toHaveTextContent(`!${(Regular.args as BoardsSectionProps).boards[i].slug}`);
+    });
   });
   
-  //Not sure if this should be it's own test or part of the previous test
   test("Board menu items link to boards", async () => {
     render(<Regular />);
-  
-    //TODO: fill this
+    const actionReturn = jest.fn();
+    mocked(action).mockReturnValue(actionReturn);
+    
+    const boards = screen.getAllByRole("link");
+    for (const board of boards) {
+      fireEvent.click(board);
+      await waitFor(() => {
+        expect(action).toBeCalledWith("#slug");
```

Comment:
Add a check that there is also a `href` in the link here. This is important for boards because we want people to be able to open the board in a new window if they want.

Theoretically it would also be nice to make the actions have the name of the board rather than the generic `#slug`, but don't waste time on it unless you're feeling like it.

Reply:
How do I query for the presence of `href`? I guess I'm not clear enough on exactly what these mocked actions are producing in terms of things that can be queried. Will `expect(board)toHaveAttribute("href")` work?

Reply:
If you remind me tomorrow (or leave a ping in the chat so I don't forget), I'll write a quick summary on mocking actions. 

Anyway, for this case, `expect(board).toHaveAttribute("href", "expected_value")` works.

I'd also suggest naming the variable `for (const boardLink of boardLinks)` so it's clear it's the links themselves and not the general board objects where the args are defined. This doesn't matter much in such small tests, but it's good to be descriptive with variable names.

Reply:
Sounds good